llvm/test/MC/SystemZ/insn-good-z14.s

```diff
@@ -108,33 +112,39 @@
 
 #CHECK:	bic	4, 0(%r7)		# encoding: [0xe3,0x40,0x70,0x00,0x00,0x47]
 #CHECK:	bil	0(%r15)			# encoding: [0xe3,0x40,0xf0,0x00,0x00,0x47]
+#CHECK:	bim	0(%r13)			# encoding: [0xe3,0x40,0xd0,0x00,0x00,0x47]
 
 	bic	4, 0(%r7)
 	bil	0(%r15)
+	bim	0(%r13)
 
 #CHECK:	bic	5, 0(%r7)		# encoding: [0xe3,0x50,0x70,0x00,0x00,0x47]
-#CHECK:	binhe	0(%r15)			# encoding: [0xe3,0x50,0xf0,0x00,0x00,0x47]
+#CHECK:	binhe	0(%r15)		# encoding: [0xe3,0x50,0xf0,0x00,0x00,0x47]
```

Comment:
See above w.r.t. spaces.  Here this may be a tab vs. spaces issue?

Reply:
NKind of. In this case, my editor assumed a tab was 4 spaces long, while this assumes a tab is 8 spaces long. I've reconfigured my editor for 8 spaces, and have realigned the tabs in this section.